site/docs/components/SwapComponents.tsx

```diff
@@ -109,7 +108,6 @@ export default function SwapComponents() {
               type="to"
             />
             <SwapButton onSubmit={onSubmit} />
-            <SwapMessage />
```

Comment:
Without exposing the component, than will be harder for the developer to customize the style.

Reply:
yeah that is a good point, what do you think @kyhyco ?

Reply:
sgtm